ChatQnA/tests/_test_chatqna_pinecone_on_xeon.sh

```diff
@@ -0,0 +1,232 @@
+#!/bin/bash
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+echo "IMAGE_REPO=${IMAGE_REPO}"
+
+WORKPATH=$(dirname "$PWD")
+LOG_PATH="$WORKPATH/tests"
+ip_address=$(hostname -I | awk '{print $1}')
+
+function build_docker_images() {
+    cd $WORKPATH
+    git clone https://github.com/opea-project/GenAIComps.git
+    cd GenAIComps
+
+    docker build -t opea/embedding-tei:latest -f comps/embeddings/langchain/docker/Dockerfile .
```

Comment:
https://github.com/opea-project/GenAIExamples/blob/558ea3bb7fc915a0622aaf13c76cf52303fee164/ChatQnA/tests/test_compose_on_xeon.sh#L23, use compose build to build images. So we can define the repo and tag for the test image. This is used by release test.    
IMAGE_REPO=${IMAGE_REPO:-"opea"}
IMAGE_TAG=${IMAGE_TAG:-"latest"}